src/hooks/useSubaccount.tsx

```diff
@@ -620,10 +620,6 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
 
         if (success) {
           onSuccess?.();
-
```

Comment:
nit: can we also add the same comment about succesfful cancelation from above here? (an old pattern I'm partial to is linking comments with a hashtag i.e. #ShortTermOrderCancellationIndexerFallback - so that you don't have to copy paste the whole thing - and we'd use a double hashtag `##` for the initial definition).